### ISSUE TITLE
I want to handling failure of pecimistic lock especially

### ISSUE THREAD

**Issue by ymaegawa** [future-architect/uroborosql#95]
I want to handling failure of pecimistic lock especially.

```Java
sqlConfig.getSqlAgentFactory().setSqlRetryCodeList(Arrays.asList("54", "30006"));
```

```Java
try {
    agent.update("example/insert_product").param("product_id", 1).retry(5, 20).count();
} catch (PecimistickLockException e) { // explicit exception
    ...
}
```

or

```Java
try {
    agent.update("example/insert_product").param("product_id", 1).retry(5, 20).count();
} catch (UroborosqlSQLException e) {
    if(agent.isRetryCode(e.getErrorCode())) { // some way to determine RetryCodeList
        ...
}
```

**Comment by ota-meshi** [comment]
@HidekiSugimoto189
Has this issue been resolved?

**Comment by HidekiSugimoto189** [comment]
This issue is fixed in #95.
Close this issue.
